Link to Patterns for Components and Actions from components section (#1573) (#1579)
ember-learn/guides-source: guides/release/components/component-state-and-actions.md

}
```

## Learn more

> As components are frequently used, patterns are very important to help reduce a lot of bugs and unforeseen issues. More in-depth guide can be found in the following chapter [Patterns for components.](../../in-depth-topics/patterns-for-components/)

Reviewer: The meaning of the word "as" wasn't clear to me. I think you meant to use the word to mean "since"?

To remove ambiguity, what do you think of:

```suggestion
You will frequently create components in an app. Establishing patterns early can help reduce bugs and unforeseen issues. Learn more from the chapter [Patterns for Components](../../in-depth-topics/patterns-for-components/).
```

I removed "very" and "a lot of" so that the sentence may sound neutral.

> And actions are JavaScript method that you can use in template. To know more about how actions can be used and their patterns, check out the following chapter [Patterns for actions.](../../in-depth-topics/patterns-for-actions/)

Reviewer: ```suggestion
Actions are JavaScript methods that you can call from a template. Find out how you can use actions from the chapter [Patterns for Actions](../../in-depth-topics/patterns-for-actions/).
```
